helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -9,12 +9,14 @@ extensions:
   {{- if and (eq (include "splunk-otel-collector.logsEnabled" .) "true") (eq .Values.logsEngine "otel") }}
   file_storage:
     directory: {{ .Values.logsCollection.checkpointPath }}
+    fsync: {{ .Values.splunkPlatform.fsyncEnabled }}
```

Comment:
Can we set this option only if it has been set explicitly by the user?

Reply:
We can put this inside an "if" block, is there any upside to that? Since this flag is already set to `false` by default here:
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/extension/storage/filestorage/factory.go

Reply:
That can change - it will change if I have anything to do with it, and we find that performance is not that impacting.

Reply:
Ok, changed it to respect the default value. I can see that there is a test failing, but it doesn't seem to be connected to changes here, and other PRs are also failing, so is that ok?

Reply:
it's not ok, please help fix that test :) but for the purpose of this PR, it won't block merging.